src/Configuration/Http/Requests/ConfigureRequest.php

```diff
@@ -5,15 +5,22 @@
 namespace MyParcelCom\Integration\Configuration\Http\Requests;
 
 use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Arr;
 
 class ConfigureRequest extends FormRequest
 {
     public function rules(): array
     {
         return [
-            'data.properties'         => 'array',
-            'data.properties.*.name'  => 'required|string',
-            'data.properties.*.value' => 'required',
+            'data' => 'array',
```

Comment:
Perhaps we can say now that it's required. It is, isn't it? We a configure request is issued there is def. data
```suggestion
            'data' => 'required|array',
```